suplemon/modules/config.py

```diff
@@ -30,6 +24,15 @@ def run(self, app, editor, args):
                     app.new_file(path)
                 app.switch_to_file(app.last_file_index())
 
+    def open_read_only(self, app, path):
```

Comment:
As future proofing for user configs, we should prob make this `open` with a `read_only=False` kwarg